install/configuring_inventory_file.adoc

```diff
@@ -746,6 +746,12 @@ openshift_cockpit_deployer_version=<cockpit_image_tag>
 |Variable |Purpose
 |`openshift_cockpit_deployer_prefix`
 |Specify the URL and path to the directory where the image is located.
+ifdef::openshift-origin[]
+The default value is `kubernetes` so the value is `cockpit/kubernetes:latest`. This differs from several other images that have the `origin-` prefix.
```

Comment:
s/is \`kubernetes\` /is \`kubernetes\`, 
s/differs/format differs
I might s/images/image paths